apps/web/package.json

```diff
@@ -14,7 +14,7 @@
     "@didtools/pkh-ethereum": "^0.1.0",
     "@lens-protocol/client": "^0.1.0",
     "@lens-protocol/widgets-react": "^0.0.10",
-    "@lit-protocol/lit-node-client": "^2.1.123",
+    "@lit-protocol/lit-node-client": "^2.2.10",
```

Comment:
@ericzakariasson I see the build is failing. We need to update the patch?

Reply:
yes, removed!